docs/getting-started.md

```diff
@@ -51,3 +51,34 @@ The username is your GitHub user name and the credential is your password. If yo
 ## Deploy your first Application
 
 Follow the instructions in the [Apache Kafka example](/docs/examples/apache-kafka/) to deploy a Kafka cluster along with its dependency Zookeeper.
+
+# Getting Started for Ops Team Members
```

Comment:
so why are the instructions and command I added recently not good enough for ops? 🤔 that was directly targeted to fix #249 

I think this is too detailed and too manual for ops. I would much rather run `make deploy | kubectl apply -` then follow 5 manual steps

I like the separation of readme to two sections though

I guess we just don't agree on how should the docs for ops look like :)

Reply:
Also I would argue ops will never run in minikube. That should go to the development part

Reply:
@alenkacz it isn't clear if you read #249 the title of that issue and your proposed solution do not match